src/Hopper/Internal/LoweredCore/EvalClosureConvertedANF.hs

```diff
@@ -174,25 +194,30 @@ localEnvLookup
 localEnvLookup env controlStack var@(LocalNamelessVar depth (BinderSlot slot))
   = go env depth
   where
-    go :: EnvStackCC -> Word32  -> EvalCC c s Ref
-    go EnvEmptyCC _ = trace "here0" $ throwEvalError (\n ->
+    go :: EnvStackCC -> Word32 -> EvalCC c s Ref
+    go EnvEmptyCC n = throwEvalError (\n ->
       BadVariableCC (LocalVar var) controlStack (InterpreterStepCC n))
```

Comment:
change 
```haskell
 go EnvEmptyCC n = throwEvalError (\n ->
        BadVariableCC (LocalVar var) controlStack (InterpreterStepCC n))		       
```
to 
```haskell
 go EnvEmptyCC _varDepth = throwEvalError (\step ->
        BadVariableCC (LocalVar var) controlStack (InterpreterStepCC step))		       
```
this is both to make the arg to go a bit more clear, AND to avoid the shadowing warning